docs/collaboration.md

```diff
@@ -0,0 +1,66 @@
+# Collaborative Publishing with Posit Publisher
+
+[Posit Publisher](https://github.com/posit-dev/publisher)—the extension to
+publish content to Posit Connect using
+[Visual Studio Code](https://code.visualstudio.com/) and
+[Positron](https://positron.posit.co/)—enables seemless collaboration. This
+workflow guide shows how to collaborate with others on a piece of content.
+
+## Initial setup
+
+To ensure that others can collaborate without issue the deployment and
+configuration files should be available.
+
+That means including them in the repository for the project or including them
+in the `files` attribute in the configuration so they are available in the
+content bundle.
```

Comment:
With my earlier comment in mind, maybe start with a publisher creating a project that is not being collaborated on and then take the next steps to make sure it can be collaborated on, then switch to the collaborator's tasks?

Reply:
That is interesting. I don't want to repeat too much that is in the ["Publishing from VS Code or Positron" document](https://docs.posit.co/connect/user/publishing-positron-vscode/) but I do like the idea of starting from the point of view of setting up a project for collaboration.